tilp/trunk/build/scripts/install_tilp.sh

```diff
@@ -209,8 +209,8 @@ listdeps() {
     echo "Arch Linux 2015.06.01 + upgrades:"
     echo -e "    pacman -S git gcc make pkgconfig autoconf automake libtool glib2 zlib libusb gtk2 libglade gettext bison flex groff texinfo xdg-utils libarchive intltool xz\n"
     echo "MacOS X:"
-    echo -e "    brew install gettext libarchive autoconf automake libtool glib lzlib libusb gtk+ libglade sdl bison flex texinfo intltool"
-    echo -e "    brew link --force libarchive gettext (you can use 'brew unlink' later)."
+    echo -e "    brew install gettext libarchive autoconf automake libtool glib lzlib libusb gtk+ libglade sdl bison flex texinfo libiconv intltool"
```

Comment:
This change was not actually needed, but heh. May get useful for static linkage later.